C_CPP/readme.md

```diff
@@ -1 +0,0 @@
-
```

Comment:
Add relative link C_CPP/Fundamentals/Introduction to C/readme.md

Reply:
Sir there is an alrdy existing link with the name Introduction To C (documentation) in your mentioned path, but I've made a video contribution, for the same name i.e. Introduction To C. So I wanted to ask do I need to add a relative link to the alrdy existing one or make a new one?